index.js

```diff
@@ -10,9 +10,8 @@ const Resizer = ({ array, updateCallback }) => {
       const newCells = Array(n - array[0].length).fill(0);
       array = array.map((r) => r.concat(newCells));
     }
-    if (n < array[0].length) {
-      array = array.map((r) => r.slice(0, n - array[0].length));
-    }
+
+    if (n < array[0].length) array = array.map((r) => r.slice(0, n - array[0].length));
```

Comment:
This line is too long, so prettier (& most linters) will break it up over two lines anyway.
Once an `if` statement is more than one line, it's easier to read (and easier to change in the future) if the brackets are there.